frontend/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "node": "14",
-    "npm": "6.13.4"
+    "node": "18",
+    "npm": "8.19.2"
```

Comment:
Also wondering here if we want to use the `^` to allow patch upgrades.

Reply:
I'm not really even sure why we have npm defined here since we don't use it on this project? 
But for the node version, if you just specify 18 it will accept any 18 version (currently 18.12)